### ISSUE TITLE
GFF feature rendering improvements

### ISSUE THREAD

**Issue by rosema62** [igvteam/igv#1243]
It would be good if desktop IGV could render features in a GFF file in specified colors depending on the feature type.  It would also be good if a feature label could be directly visible on the feature without having to click on it.

Thanks

**Comment by jrobinso** [comment]
To see a feature label set a "Name" property in column 9.

**Comment by rosema62** [comment]
Ahh.  My GFF's ids are labeled "label" not "Name".  Thanks, that did the trick.

**Comment by jrobinso** [comment]
Good, glad that worked, its part of the GFF specification

```
**Name A**   display name for the feature
```

**Comment by jrobinso** [comment]
I'm going to re-open and rename this for the color suggestion

**Comment by malcook** [comment]
Would you consider to generalize it to "Color feature by GFF attribute" (including those possibly encdoded in bed/bigbed Name field using #gffTags  )???

**Comment by jrobinso** [comment]
Yes, I think it could just be "color by attribute" and work like color-by-sam-tag for alignments, you type in the attribute name.   Certain standard column names, like "type", maybe "source", could pose a conflict if those attributes were also present in column 9.    However it would work reasonably well for most files.
